src/main/resources/com/mabl/integration/jenkins/MablStepBuilder/help-applicationId.html

```diff
@@ -1,3 +1,3 @@
 <div>
-    mabl application identifier. Hint: URL component ending in "-a"
+    Input a valid ApiKey to populate a dropdown of application names to select from.
```

Comment:
Sorry, I blame my English teacher. Can we remove the dangling preposition and do something like `...to populate the application name dropdown` for applications and environments?